Dockerfile

```diff
@@ -89,6 +89,14 @@ RUN curl -L -O https://github.com/cnoe-io/cnoe-cli/releases/download/v0.1.0/cnoe
 COPY ./cnoe-wrapper.sh /usr/bin/cnoe
 RUN chmod +x /usr/bin/cnoe
 
+RUN apt-get update && apt-get install -y python3 python3-pip python3-venv
```

Comment:
We do a lot of apt work; can we merge these a bit? Also maybe delete the cache before we finish

Reply:
I merged it with the kubectl install step above. Cache is mounted so the end image doesn't have any.

Reply:
Ah, I see that now, thanks!